provider/rfc2136.go

```diff
@@ -41,6 +41,7 @@ type rfc2136Provider struct {
 	tsigSecretAlg string
 	insecure      bool
 	axfr          bool
+	minTTLSeconds int
```

Comment:
why not using int64 directly? so you can drop those lines:
```
var ttl = int64(r.minTTLSeconds)
	if ep.RecordTTL.IsConfigured() && int64(ep.RecordTTL) > ttl {
		ttl = int64(ep.RecordTTL)
	}
```

Reply:
I can use int64 but still I need to explicitly cast TTL to int64. So I can't drop those castings.